diff --git a/src/unstructured_client/general.py b/src/unstructured_client/general.py
index 3b8af5f..05f0962 100644
--- a/src/unstructured_client/general.py
+++ b/src/unstructured_client/general.py
@@ -11,8 +11,9 @@ class General(BaseSDK):
     
     
     def partition(
-        self, *,
+        self,
         request: Union[operations.PartitionRequest, operations.PartitionRequestTypedDict],
+        *,
         retries: OptionalNullable[utils.RetryConfig] = UNSET,
         server_url: Optional[str] = None,
         timeout_ms: Optional[int] = None,
diff --git a/src/unstructured_client/models/operations/partition.py b/src/unstructured_client/models/operations/partition.py
index 465bc3a..a537a13 100644
--- a/src/unstructured_client/models/operations/partition.py
+++ b/src/unstructured_client/models/operations/partition.py
@@ -19,6 +19,9 @@ class PartitionRequestTypedDict(TypedDict):
 class PartitionRequest(BaseModel):
     partition_parameters: Annotated[shared_partition_parameters.PartitionParameters, FieldMetadata(request=RequestMetadata(media_type="multipart/form-data"))]
     unstructured_api_key: Annotated[OptionalNullable[str], pydantic.Field(alias="unstructured-api-key"), FieldMetadata(header=HeaderMetadata(style="simple", explode=False))] = UNSET
+
+    def __init__(self, partition_parameters, **kwargs):
+        BaseModel.__init__(self, partition_parameters=partition_parameters, **kwargs)
     
     @model_serializer(mode="wrap")
     def serialize_model(self, handler):
